book.rb

```diff
@@ -0,0 +1,14 @@
+# book.rb
+
+#
+# Purpose: Book class
+#
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] to make the many to many relationship work Getters and Setters for @rentals should be present
- [ ] A method for adding rentals should also be present. please see this [Link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) to the example in a previous lesson.

Reply:
Thank you. changes done!